lib/ir_pagerduty.rb

```diff
@@ -0,0 +1,19 @@
+require 'pagerduty/full'
+
+class IRPagerduty < PagerDuty::Full
+  def initialize
+    @apikey = SupportApp.pager_duty_token
+    @subdomain = SupportApp.pager_duty_subdomain
+  end
+
+  def fetch_json(path, params)
+    res = self.api_call(path, params)
+
+    case res
+      when Net::HTTPSuccess
+        JSON.parse(res.body)
+      else
+        res.error!
+    end
+  end
```

Comment:
We've got a bit of duplication here with what we're doing for displaying alerts:

https://github.com/ministryofjustice/2nd-line-support/blob/04196b6d1c4932e00ddb1bfe448beba0e8fd24ad/services/pagerduty_alerts.rb#L38-L48

I'm not quite sure what the fix is but it seems slightly wrong to create a dedicated class *just* for this. Maybe we make these private methods in the services/people.rb People class?